### ISSUE TITLE
FTorch having issue with torch.nn.Sigmoid (possibly an edge case)

### ISSUE THREAD

**Issue by jiarong-wu** [leap-stc/Integration_team#8]
During implementation of my NN air-sea flux algorithm, I observed that the code would throw an floating point error, which was fixed by manually code a sigmoid function in place of the torch.nn.Sigmoid function.
